### ISSUE TITLE
output of student become larger as training progresses

### ISSUE THREAD

**Issue by sxwenny** [Westlake-AGI-Lab/Distill-Any-Depth#35]
Thank you for your excellent work！
When I use Large to distill Small, the output value of Small will become larger and larger as the training progresses.
Do you know why this is?

**Comment by shuiyued** [comment]
Hello,

Sorry for later reply.

Regarding the issue you mentioned, I’d suggest trying the following debugging approach:

1) Load the pretrained weights of the smaller model.
2) Monitor the loss values iteration by iteration. This might help identify whether the issue lies in the supervision signal.
